TrafficCapture/trafficCaptureProxyServer/src/test/java/org/opensearch/migrations/trafficcapture/proxyserver/KafkaConfigurationCaptureProxyTest.java

```diff
@@ -129,17 +129,30 @@ public void testCaptureProxyWithKafkaImpairedDoesNotAffectRequest_proxysRequest(
             captureProxy.start();
             final int numberOfTests = 20;
 
-            // Performance is different for first few calls so throw them away
-            assertBasicCalls(captureProxy, 3);
+            // Performance is different for first call so throw away
+            assertBasicCalls(captureProxy, 1);
 
-            var averageBaselineDuration = assertBasicCalls(captureProxy, numberOfTests);
+            var totalBaselineDuration = Duration.ZERO;
+            var totalImpairedDuration = Duration.ZERO;
 
-            failureMode.apply(kafkaProxy);
+            for (int i = 0; i < numberOfTests; i++) {
+                // Run baseline case
+                totalBaselineDuration = totalBaselineDuration.plus(assertBasicCalls(captureProxy, 1));
+
+                // Apply failure mode
+                failureMode.apply(kafkaProxy);
+
+                // Run impaired case
+                totalImpairedDuration = totalImpairedDuration.plus(assertBasicCalls(captureProxy, 1));
+
+                // Reset failure mode
+                removeAllFailureModes(kafkaProxy);
+            }
 
-            // Calculate average duration of impaired calls
-            var averageImpairedDuration = assertBasicCalls(captureProxy, numberOfTests);
+            var averageBaselineDuration = totalBaselineDuration.dividedBy(numberOfTests);
+            var averageImpairedDuration = totalImpairedDuration.dividedBy(numberOfTests);
 
-            long acceptableDifference = Duration.ofMillis(25).toMillis();
+            long acceptableDifference = Duration.ofMillis(20).toMillis();
```

Comment:
why did this get tightened?

Reply:
I made this more resilient with the other changes so this can be tightened.